### DISCUSSION TITLE
How do I get in contact with the person(s) packaging a specific app for homebrew?

### DISCUSSION THREAD

**Discussion by foss4ever** [Homebrew/discussions#4904] [Everyday usage]
### Output of `brew config`

```shell
n/a
```

### Output of `brew doctor`

```shell
n/a
```

### Description of issue

As the title says:
How do I get in contact with the person(s) packaging a specific app for homebrew?

I need to get a message across from the maintainers of openfortivpn to the packagers of this same application in Homebrew:

_Packagers should build openfortivpn with configure --enable-legacy-pppd, required for platforms still shipping pppd < 2.5.0._

Any help is much appreciated.

**Comment by SMillerDev** [comment]
You don't, no such people exist. Homebrew is maintained by the public. If you want something to change in Homebrew, https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request is the best place to look.

**Reply by foss4ever** [reply]
Ok.. I'm not terribly familiar with the Homebrew ecosystem other than installing various applications that are not otherwise available.
Any chance you could provide a bit more guidance to a novice as to how I can get the fix suggested by openfortivpn maintainers introduced to the homebrew-offered version ?
Should I maybe post an issue at https://github.com/Homebrew/homebrew-cask/issues or is that besides the point ?

**Reply by SMillerDev** [reply]
Seeing how you're talking about building something from source it's definitely not a cask issue since those are only pre compiled binaries.
